feat (docket): add modal to buy links

albertisfu opened the pull request:
Fixes: #1352

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/freelawproject/courtlistener?pullRequest=1869) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/freelawproject/courtlistener?pullRequest=1869) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/freelawproject/courtlistener?pullRequest=1869) it.</sub>

albertisfu commented:
Added install recap modal to more buttons:

- Added in rd_download_button.html if recap document is available or not
- Added in pacer_links_dropdown.html View on Pacer main button and toggle dropdown buttons
- Added in docket_li.html for View on Pacer action in tag template
- Added in view_docket.html for Buy Docket on Pacer button when no docket entries available

mlissner commented on cl/opinion_page/templates/includes/buy_pacer_modal.html:
```diff
@@ -0,0 +1,32 @@
+      <div class="modal-footer">
+        <p>
+          <a href="https://free.law/recap/" target="_blank"
+             class="btn btn-default">Install RECAP</a>
```
```suggestion
             class="btn btn-default">Learn More</a>
```

mlissner reviewed (approved):
This looks good to me. Merging after making one more tweak to the modal itself.

mlissner commented:
The tests are having a minor fit because they don't have the S3 rights they need, but this will soon be fixed by @satsuki-chan, I think.

Thank you Alberto for your first contribution!
